app/models/user.rb

```diff
@@ -159,6 +159,22 @@ def can_moderate?
     admin? || moderator?
   end
 
+  # User status
+  # 0: banned
+  # 5: moderated
+  def mark_as!(new_status)
```

Comment:
hi @seafr,  changes look good, I would agree with 'normal' as status as for comments we use 'normal' only